.github/workflows/continuous-integration.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
           git config user.email "actions@github.com"
           git config user.name "GitHub Actions"
           git add .
-          git commit -m "style: format files with Prettier"
+          git commit -m "style: format files with Prettier" | true # ignore error if there are no changes to commit
```

Comment:
This seems fine, though it feels like a hack. Is there a way we can exit early if nothing has changed (i.e. if `npm run format` passes)?

Reply:
Ah, just saw your comments at https://github.com/nock/nock/issues/1883#issuecomment-586748145. Seem fine for now.